src/client.py

```diff
@@ -0,0 +1,32 @@
+"""Something client-y."""
+
+import sys
+import socket
+
+
+def client(message):
+
+    message += 'EOF'
```

Comment:
This is tricky. What if the message coming in from the command line is something like "test_meEOF I keep working even with the EOF in there."